mnefun/_mnefun.py

```diff
@@ -1417,18 +1418,17 @@ def save_epochs(p, subjects, in_names, in_numbers, analyses, out_names,
             # now make evoked for each out type
             evokeds = list()
             for name in names:
-                if p.weighting:
-                    #TODO give access to _chpi_snr_epochs args?
+                if p.average_weighting == 'chpi':
                     weights = _chpi_snr_epochs(e[name])
```

Comment:
@jjnurminen the input to `_chpi_snr_epochs` here is epochs that were created after movement compensation and additional processing. Does this result in accurate weights i.e., same as weights computed from raw file prior to movement compensation? That is not my understanding cc @Eric89GXL